test/run-php-linter.sh

```diff
@@ -3,7 +3,11 @@ set -euo pipefail
 
 # Run PHP -l on everything to ensure there's no syntax
 # errors.
-find docs modules htdocs php src tools -name '*.class.inc' -print0 -o -name '*.php' -print0 |xargs -0 -n1 php -l >/dev/null  
+find docs modules htdocs php src tools \
```

Comment:
Why is this file changing? It's a shell script, it shouldn't be affected by PHPCS.

Reply:
It's just a formatting change so that these commands aren't long overflowing one-liners. I'm hoping that this will make them more readable and easier to maintain